config/routes.rb

```diff
@@ -567,15 +570,15 @@ def scoped_navigation_routes(context, navigation)
         end
       end
 
-      scope ':us_state', constraints: { us_state: /az|ny/i } do
+      scope ':us_state', constraints: { us_state: Regexp.new(active_state_codes.join("|"), Regexp::IGNORECASE) } do
         resources :submission_pdfs, only: [:show], module: 'state_file/questions', path: 'questions/submission_pdfs'
         resources :federal_dependents, only: [:index, :new, :create, :edit, :update, :destroy], module: 'state_file/questions', path: 'questions/federal_dependents'
         resources :unemployment, only: [:index, :new, :create, :edit, :update, :destroy], module: 'state_file/questions', path: 'questions/unemployment'
         get "/data-import-failed", to: "state_file/state_file_pages#data_import_failed"
         get "/initiate-data-transfer", to: "state_file/questions/initiate_data_transfer#initiate_data_transfer"
       end
 
-      scope ':us_state', constraints: { us_state: /az|ny|us/i } do
+      scope ':us_state', constraints: { us_state: Regexp.new((active_state_codes + ["us"]).join("|"), Regexp::IGNORECASE) } do
```

Comment:
NIT: Maybe include a small comment above with an example of what it generates for easy future reference.

Reply:
Good call - done!